src/nagendra/CodingExam3.java

```diff
@@ -0,0 +1,21 @@
+package nagendra;
+
+public class CodingExam3 {
+
+	void geDuplicateArray(int[] input) {
+		for (int i = 0; i < input.length; i++) {
+			for (int j = i+1; j < input.length; j++ ) {
```

Comment:
As discussed,
{ 10, 12,32, 55, 32, 17, 12, 32 } try this input. code wont work as expected